test/test_unit.jl

```diff
@@ -414,9 +414,28 @@ isdir(outdir) && rm(outdir, recursive=true)
     @test Trixi.varnames(cons2mean, equations) == ("v1_mean", "v2_mean", "c_mean", "rho_mean")
   end
 
+  @testset "Euler conversion between conservative/entropy variables" begin
+    tol = 10*eps()
+    ρ,u,v,w,p = 1.0, .1, .2, .3, 2.0
+
+    cons_vars = Trixi.prim2cons([ρ,u,p],CompressibleEulerEquations1D(1.4))
+    entropy_vars = Trixi.cons2entropy(cons_vars,CompressibleEulerEquations1D(1.4))
+    @test maximum(abs.(cons_vars .- Trixi.entropy2cons(entropy_vars,CompressibleEulerEquations1D(1.4)))) < tol
```

Comment:
Usually, it's better to just use `isapprox` (or its equivalent `≈`) when testing equality of two vectors. Then, you do not need the tolerance anymore and the check could read like
```suggestion
    rho, v1, v2, v3, p = 1.0, 0.1, 0.2, 0.3, 2.0

    let equations = CompressibleEulerEquations1D(1.4)
      cons_vars = prim2cons(SVector(rho, v1, p),equations)
      entropy_vars = cons2entropy(cons_vars, equations)
      @test cons_vars ≈ entropy2cons(entropy_vars, equations)
    end
```
The other dimensions could be updated accordingly.

Reply:
Got it. I stopped using`isapprox` because I didn't like their handling of near zero quantities, but it should be fine here.